packages/quantic/force-app/main/default/lwc/quanticSearchBox/quanticSearchBox.js

```diff
@@ -94,6 +94,9 @@ export default class QuanticSearchBox extends LightningElement {
 
   setHighlighted() {
     const options = this.getSuggestionElements();
```

Comment:
I would use the term `suggestions` instead of `options` to avoid confusion with engine and/or component options.

Reply:
nice find! sure.